runtime/bcverify/staticverify.c

```diff
@@ -122,7 +122,12 @@ buildInstructionMap (J9CfrClassFile * classfile, J9CfrAttributeCode * code, U_8
 			break;
 
 		default:
-			if (bc > CFR_BC_jsr_w) {
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+			if ((bc > CFR_BC_withfield) || (bc == CFR_BC_breakpoint))
+#else /* J9VM_OPT_VALHALLA_VALUE_TYPES */
+			if (bc > CFR_BC_jsr_w)
```

Comment:
I don't see `CFR_BC_breakpoint` is checked here. Is this something that we missed ?

Reply:
Here we're checking which BC is illegal. Anything greater than max is illegal. According to Tobi, `CFR_BC_breakpoint` must also be illegal.
`CFR_BC_breakpoint` is greater than `CFR_BC_jsr_w` so it is already covered by the existing code. However if BC's greater than `CFR_BC_withfield` are illegal, then `CFR_BC_breakpoint` would go through.

Reply:
OK. Thx.